week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,55 @@
 /** Write your CSS in here **/
+body{
+    background: rgb(245, 171, 33);
+  }
+  h3{
+      color:  rgb(245, 171, 33);
+      font-size: 24px;
+      text-align: end;
+  }
+  p{
+  
+    margin:100px 90px;
+    padding:30px 30px;
+    color:rgb(245, 171, 33);
+    align-self: flex-end;
```

Comment:
`align-self` is a flexbox rule, so it will only work is you also use `display: flex`